### DISCUSSION TITLE
solid potential in electrolyte current formulation

### DISCUSSION THREAD

**Discussion by dion-w** [pybamm-team/PyBaMM#1819] [Q&A] [answered]
Hello everyone,
i have a question concerning the equation for the electrolyte current in the submodel {"electrolyte conductivity": "full", "surface form": "differential"}. The equation for electrolyte current in `get_coupled_variables`  in the file `pybamm/models/submodels/electrolyte_conductivity/surface_potential_form/full_surface_form_conductivity.py` (**line 59**) reads like this:
```
i_e = conductivity * (
                ((1 + param.Theta * T) * param.chi(c_e, T) / c_e) * pybamm.grad(c_e)
                + pybamm.grad(delta_phi)
                + i_boundary_cc / sigma_eff
            )
```
My question now is: Why do you use delta_phi in this equation? Normally for conduction in the electrolyte you would use the electrolyte potential phi_e and not delta_phi when treating a capacitor-like double layer (as seen [here](https://doi.org/10.1149/2.0301711jes) and [here](https://doi.org/10.1149/1945-7111/ab67c7)). I sort of get, that you want to use delta_phi, so you only need to solve the model for c_s, c_e and delta_phi but i don't get the physical motivation, or does the "i_boundary_cc/sigma_eff" in the i_e equation above somehow cancel out the contribution of grad(phi_s)? I see some stuff in the boundary conditions, but i gues it would cancel stuff out only on the boundaries and not in between? 

Thanks in advance for your help.

**Comment by dion-w** [comment]
By doing a comparison between the i_e from the double layer fromulation and the i_e without a double layer i found out that by comparison both equations differ in a term of <img src="https://render.githubusercontent.com/render/math?math=\frac{\partial \phi_{s}}{\partial t}=\frac{i_{e}-i_{cc}}{\sigma_{eff}}">. This means when including this term in the formulaton with double layer one gets the equation without double layer. So to use the double layer in dependence of <img src="https://render.githubusercontent.com/render/math?math=\frac{\partial \Delta \phi_{s}}{\partial t}"> there needs to be an additional term as well as an updated prefactor (which includes the conductivities). Can anybody confirm or deny this? Is there any publication about this? I found nothing else than the usual papers when using `pybamm.print_citations` at the end of my script.

**Comment by valentinsulzer** [comment, marked as answer]
This is the paper: https://iopscience.iop.org/article/10.1149/2.0301910jes/pdf
See equations 47 and 48. The variable in the time derivative is delta phi

**Reply by dion-w** [reply]
Thanks a lot!
